SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -0,0 +1,329 @@
+#!/bin/bash
+
+# Author: Rodrigo Sobral 2024
+# Copyright CERN
+# Here the script is used to create a virtual environment and install the packages from a provided requirements file.
+
+_log () {
+    if [ "$*" == "ERROR:"* ] || [ "$*" == "WARNING:"* ] || [ "${JUPYTER_DOCKER_STACKS_QUIET}" == "" ]; then
+        echo "$@"
+    fi
+}
+
+# if [ -f "/home/$USER/.bashrc" ]; then
+#     source /home/$USER/.bashrc
+# fi
+# CUSTOMENVS_LIMIT=1
+ACCPY_BASE=$ACCPY_PATH/base
+ACCPY_ALL_VERSIONS_STR="?"
+PYTHON_DEFAULT_PATH=$(which python)
+if [ -d "$ACCPY_PATH" ]; then
+    ACCPY_ALL_VERSIONS=$(ls -tr $ACCPY_BASE)
+    ACCPY_ALL_VERSIONS_STR=$(echo $ACCPY_ALL_VERSIONS | tr ' ' ', ')
+fi
+
+
+# Function to print the help page
+print_help() {
+    _log "Usage: makenv --env/-e NAME --req/-r REQUIREMENTS [--accpy ACCPY_VERSION] [--python PATH] [--clear/-c] [--help/-h]"
+    _log "Options:"
+    _log "  -e, --env NAME              Name of the custom virtual environment (mandatory)"
+    _log "  -r, --req REQUIREMENTS      Path to requirements.txt file or http link for a public repository (mandatory)"
+    _log "  -c, --clear                 Clear the current virtual environment, if it exists"
+    _log "  -h, --help                  Print this help page"
+    _log "  --accpy VERSION             Version of Acc-Py to be used (options: ${ACCPY_ALL_VERSIONS_STR})"
+    _log "  --python PATH               Path to the Python interpreter to be used (default: ${PYTHON_DEFAULT_PATH})"
```

Comment:
I'm thinking we could remove this option for now. For the venvs we will use either the acc-py Python or the one in our image.
You can keep a version of this script in another branch of yours that still has this feature, in case we want to rescue it in the future.